tests_integration/__tests__/__snapshots__/config-resolution.js.snap

```diff
@@ -58,7 +58,7 @@ exports[`accepts configuration from --config 1`] = `
 "/* eslint-disable */
 
 function f() {
-  console.log(\\"should have tab width 8\\")
+        console.log(\\"should have tab width 8\\");
```

Comment:
Why did this change?

Reply:
I'm still investigating, and there are still some `stdin` needs to be mocked (`options.input`).